scripts/excfs_gdas_vrfyfits.sh

```diff
@@ -83,8 +82,8 @@ $BUFRPOSTSH $sig1 $CNVS $PRPI $PRPO $CDATE
 [ $SAVEPREP = YES          ] && cp $PRPO $ARCDIR/$(basename $PRPO).06.00
 [ $SAVEPREP = YES          ] && cp $PRPI $ARCDIR
 
-$FITSSH     $CDATE $PRPO $COMLOX $DATA 06 00           
-$HORZSH     $CDATE $PRPO $COMLOX $DATA anl 2> horizout
+$FITSSH     $CDATE $PRPO $COMOUT_FITX $DATA 06 00
+$HORZSH     $CDATE $PRPO $COMOUT_FITX $DATA anl 2> horizout
```

Comment:
Nitpick: Should probably use `{}` around variables.

Reply:
I agree but want to leave that for another PR. Didn't want to muddy this one with tons of shellcheck fixes. :)